ethereum/core/src/test/java/org/hyperledger/besu/ethereum/bonsai/RollingImport.java

```diff
@@ -61,11 +60,7 @@ public static void main(final String[] arg) throws IOException {
     final SegmentedInMemoryKeyValueStorage worldStateStorage =
         (SegmentedInMemoryKeyValueStorage)
             provider.getStorageBySegmentIdentifiers(
-                List.of(
-                    ACCOUNT_INFO_STATE,
-                    CODE_STORAGE,
-                    ACCOUNT_STORAGE_STORAGE,
-                    TRIE_BRANCH_STORAGE));
+                List.of(ACCOUNT_INFO_STATE, ACCOUNT_STORAGE_STORAGE, TRIE_BRANCH_STORAGE));
```

Comment:
I was going to call out that not including code storage in world state storage would make this test fail for any log rolling that included code.  But I don't know whether this class is even useful anymore.  It looks like it was part of an early prototype PR but isn't used anywhere anymore.

Reply:
Wasn't sure what this was for. Think it's an old testing tool? It's in the test package so wasn't worried about not including the code storage in world state storage.